drill-yarn/src/main/java/org/apache/drill/yarn/appMaster/http/WebServer.java

```diff
@@ -306,18 +304,18 @@ private ConstraintSecurityHandler createSecurityHandler() {
   }
 
   /**
-   * It creates A {@link SessionHandler} which contains a {@link HashSessionManager}
+   * It creates A {@link SessionHandler}
```

Comment:
Sorry to make you do this twice. The DoY web server was pretty much a copy/paste clone of the Drill exec version. The goal was to factor out a common set of code, but that never quite happened...